website/docs/docs/upgrading-from-v3/README.md

```diff
@@ -50,7 +50,7 @@ for more information on this API.
 
 Closes #451 Closes #439
 
-### Adds new `unstable_effectOn` model API
+### Adds new `effectOn` model API
 
 > **\*Note:** this is an experimental API. We are pre-releasing it to allow for
```

Comment:
Actually, just checked out the branch and ran a search for "unstable" and noticed this section that we should probably remove. Otherwise, LGTM

Reply:
Just pushed the change.